_inc/client/plans/single-product-backup.jsx

```diff
@@ -238,19 +274,22 @@ class SingleProductBackupBody extends React.Component {
 						/>
 					) ) }
 				</div>
-				{ upgradeLinks &&
-					upgradeLinks[ selectedBackupType ] &&
-					upgradeTitles[ selectedBackupType ] && (
-						<div className="single-product-backup__upgrade-button-container">
-							<Button
-								href={ upgradeLinks[ selectedBackupType ] }
-								onClick={ this.handleUpgradeButtonClick( selectedBackupType ) }
-								primary
-							>
-								{ upgradeTitles[ selectedBackupType ] }
-							</Button>
-						</div>
-					) }
+				<p>
+					<em>
+						{ __( 'You are saving $40 by paying yearly. {{a}}Switch to monthly{{/a}}', {
```

Comment:
Is it okay to have the `$40` value be static here? Wouldn't we want this to be dynamic so it is always accurate, regardless of ongoing sales or currency?

Reply:
FWIW, if we take Jetpack Backup Real-time, here's how we calculate the savings in various locations (I'm using Euros for the example, but it's the same for any currency, and you get the picture):

* Monthly price - €20 per month
* Yearly price: €200 per year
* X = Price for a year, when paying monthly - 12 * €20 = €240
* Y = Price for a year, when paying yearly - €200
* Savings when paying yearly: X - Y = €240 - €200 = €40

So we can totally keep it dynamic.

This calculation already exists in other locations, so ideally we shouldn't be duplicating it.

Reply:
I adjusted things to calculate the difference and set text appropriately in 61f85718cf3a715b1b85a13df7ab4883095d6c67. Sorry for the large and vague commit :)